instrumentation/racecar/CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Release History: opentelemetry-instrumentation-racecar
 
+* Renamed messaging.kafka.message_key to messaging.kafka.message.key to match semantic conventions.
+
+* Renamed messaging.destination to messaging.destination.name to match semantic conventions.
+
+* Renamed messaging.kafka.partition to messaging.destination.partition.id to match semantic conventions.
+
+* Removed messaging.destination_kind to match semantic conventions.
+
+* Add messaging.operation.type attribute as per semantic conventions.
```

Comment:
Our release tooling will add the changelog entry. The default is based on the name of the PR. I'm happy to edit the entry to match this text, but we'll have to wait until the release phase to do so.